### ISSUE TITLE
[BUG] Built in rule TA-000001 false-positive on site config object

### ISSUE THREAD

**Issue by tonybaloney** [Azure/template-analyzer#363]
### Describe the bug

The builtin rule TA-000001 has the following rule:

```json
"anyOf": [
        {
          "resourceType": "Microsoft.Web/sites/config",
          "allOf": [
            {
              "path": "properties.detailedErrorLoggingEnabled",
              "equals": true
            },
            {
              "path": "properties.httpLoggingEnabled",
              "equals": true
            },
            {
              "path": "properties.requestTracingEnabled",
              "equals": true
            }
          ]
        },
```

But, whilst those are the property names for `properties.siteConfig` in the `Web/Sites` app, they are _not_ for the `Microsoft.Web/sites/config` resource.

This is a valid site config, but the rule is still triggering on this template-- 

```bicep
resource configLogs 'Microsoft.Web/sites/config@2022-03-01' = {
  name: 'logs'
  parent: appService
  properties: {
    applicationLogs: { fileSystem: { level: 'Verbose' } }
    detailedErrorMessages: { enabled: true }
    failedRequestsTracing: { enabled: true }
    httpLogs: { fileSystem: { enabled: true, retentionInDays: 1, retentionInMb: 35 } }
  }
  dependsOn: [configAppSettings]
}
```

### Expected behavior

Fix the base rule to use the correct paths for config objects

### Reproduction Steps

see above

**Comment by EfratMalkaMS** [comment]
Hi @tonybaloney,

Thank you for reaching out with this issue.
We are going to triage it and update back on this thread.
 
Thanks,
MSDO team

**Comment by JohnathonMohr** [comment]
Hey @tonybaloney, it looks like the gap here is between the *type* of site config (controlled by the *name* property). The properties in the rule are correct for a [Web config](https://learn.microsoft.com/en-us/azure/templates/microsoft.web/2021-02-01/sites/config-web?pivots=deployment-language-arm-template) (where `name` is "web"), but the schema is different for the [Logs config](https://learn.microsoft.com/en-us/azure/templates/microsoft.web/2021-02-01/sites/config-logs?pivots=deployment-language-arm-template) (where `name` is "logs").

As @EfratMalkaMS mentioned, we still need to triage this, but I wanted to give a quick update on initial investigation.

**Comment by EfratMalkaMS** [comment]
Hi @tonybaloney,

We're still investigating the issue, is this a blocking issue?

Thakns,
Efrat

**Comment by tonybaloney** [comment]
@EfratMalkaMS yes I have this false-positive on 23 separate projects.

**Comment by EfratMalkaMS** [comment]
Hi @tonybaloney ,

We're still working on triaging the issue and wanted to check if you could possibly mitigate the issue by suppressing the specific finding in question that would be great.

Thanks,
Efrat
